cnxarchive/search.py

```diff
@@ -37,6 +37,7 @@
                  [chr(i) for i in range(ord('a'), ord('z') + 1)])
 WILDCARD_KEYWORD = 'text'
 VALID_FILTER_KEYWORDS = ('type', 'pubYear', 'authorID', 'submitterID')
+SEARCH_SPECIAL_CHARS = '"\',:'
```

Comment:
Ah right, strip() only removes from the extremes of the string. so "" and '' and :: and ,, would all string to nothing. How about we keep that, but change the numeric limit to < 2, or remove it completely? I'm particularly concerned about single-char Chinese being useful searches. Two char are definitely useful